### PR TITLE
Feature / speakers

### PR DESCRIPTION
**Pull request by paulodiovani**
- Add speakers section
- Add speakers

### CODE REVIEW

**Review by boniattirodrigo — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -16,8 +16,8 @@
   <meta property="og:image" content="https://hacktoberfestnh.github.io/img/hacktoberfestnh.png" />
 
   <!-- Favicons -->
-  <link href="img/favicon.png" rel="icon">
-  <link href="img/apple-touch-icon.png" rel="apple-touch-icon">
+  <!-- <link href="img/favicon.png" rel="icon"> -->
+  <!-- <link href="img/apple-touch-icon.png" rel="apple-touch-icon"> -->
```
Comment (nolleto) [inline index.html:20]: Why don't you delete these commented links?

File: index.html [hunk 2]
```diff
@@ -142,108 +142,105 @@ <h3>Quando</h3>
...
+                <p>
+                  Software Engineer, Consultant, and Small Business Owner in São Paulo, Brazil.
+                  YouTuber at Akitando.
+                  THE CONF founder and organizer.
+                  OmniTrade co-founder.
+                  Codeminer 42 co-founder.
+                  Ex RubyConf Brazil Program Chairman.
+                  Founder and organizer of THE CONF.
+                  Software Craftsmanship Evangelist.
+                  Tech Speaker in many conferences.
```
Comment (nolleto) [inline index.html:190]: Why is only Akita's bio in English and the other ones are in Portuguese?

**Comment by nolleto** [comment]
Too late 😅

File: index.html (hunk 1 above)
Comment (paulodiovani) [reply index.html:20]: A lot of code is just commented to allow easier update later.
`favicon` should be changed once we have a good alternative.

File: index.html (hunk 2 above)
Comment (paulodiovani) [reply index.html:190]: I just copied from his blog. :sweat_smile: 

Feel free to translate. :wink:

**Pull request merged**